xls/modules/zstd/benchmarks/BUILD

```diff
@@ -0,0 +1,658 @@
+# Copyright 2024 The XLS Authors
```

Comment:
I think we can remove this file, since everything is commented out.

Reply:
It's temporarily commented out because of the problems with proc inlining (see https://github.com/google/xls/issues/1455). With that we are unable to generate verilog out of certain procs and check the max frequency estimate through IR benchmark or Place and Route flow.
We will enable those rules back in the current state, but please note that this will cause multiple failures during IR optimization process in the ZSTD-specific CI workflow.